client/src/index.tsx

```diff
@@ -0,0 +1,5 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import App from './components/App';
+
+ReactDOM.render(<App />, document.getElementById('root'));
```

Comment:
ReactDOM.render is [deprecated in v18](https://react.dev/blog/2022/03/08/react-18-upgrade-guide)